tasmota/tasmota_xdrv_driver/xdrv_91_zerocrossDimmer.ino

```diff
@@ -0,0 +1,235 @@
+/*
+  xdrv_91_zerocrossdimmer.ino - Zero-Cross Dimmer support for Tasmota
+
+  Copyright (C) 2023  Stefan Bode
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifdef USE_AC_ZERO_CROSS_DIMMER
+/*********************************************************************************************\
+ * Zero-Cross AC Dimmer PMM 1..xx use 
+\*********************************************************************************************/
+
+#define XDRV_91             1

```

Comment:
Take xdrv_68 as it's supposed to work for esp8266 too. Do not forget to set it to 68 too. It currently is set to 1 which is not good.